src/torrent/download_info.h

```diff
@@ -33,16 +33,18 @@ class LIBTORRENT_EXPORT DownloadInfo {
     STOPPED
   };
 
-  static constexpr int flag_open                = (1 << 0);
-  static constexpr int flag_active              = (1 << 1);
-  static constexpr int flag_accepting_new_peers = (1 << 3);
-  static constexpr int flag_accepting_seeders   = (1 << 4); // Only used during leeching.
-  static constexpr int flag_private             = (1 << 5);
-  static constexpr int flag_meta_download       = (1 << 6);
-  static constexpr int flag_pex_enabled         = (1 << 7);
-  static constexpr int flag_pex_active          = (1 << 8);
+  enum flag {
+    open                = (1 << 0),
+    active              = (1 << 1),
+    accepting_new_peers = (1 << 3),
+    accepting_seeders   = (1 << 4), // Only used during leeching.
+    priv                = (1 << 5),
```

Comment:
Why are you shortening this and def? Makes no sense.

Reply:
private is not allowed to be used.

Reply:
Then it should be a more verbose name, not something that harkens back to the mistakes of the 70's C era.